src/test/groovy/st/ratpack/auth/CachingTokenValidatorSpec.groovy

```diff
@@ -0,0 +1,46 @@
+package st.ratpack.auth
+
+import ratpack.exec.ExecResult
+import ratpack.exec.Promise
+import ratpack.test.exec.ExecHarness
+import spock.lang.Specification
+
+class CachingTokenValidatorSpec extends Specification {
+
+	ExecHarness harness
```

Comment:
You can initialise inline, and you should annotate this with `@spock.lang.AutoCleanup` so the thread pool is shut down.

See: http://ldaley.com/post/3133819681/spocks-autocleanup-feature